cypress/page/payment-step.page.ts

```diff
@@ -1,10 +1,11 @@
 class PaymentStepPage {
     private payButton: string;
     private confirmOrderMessage: string;
+    
 
     constructor() {
         this.payButton = "a.bankwire";
-        this.confirmOrderMessage = "#cart_navigation>button>span";
+        this.confirmOrderMessage = "#cart_navigation .button";
```

Comment:
1. This variable should be called something like confirmOrderButton since this selector is for the button not for the message
2. can you create a variable to store the selector used in the `confirmMessage` method.